convert.py

```diff
@@ -145,7 +152,7 @@ def _extract_paths(args: argparse.Namespace) -> list[Path]:
             "No paths have been passed as command line args."
             "No paths have been defined in the user config."
         )
-    paths = [Path(str_path) for str_path in str_paths]
+    paths = [Path(str_path).expanduser() for str_path in str_paths]
```

Comment:
Do we also need `resolve` here as in `check_if_all_paths_exist` in `files.py`?

Reply:
We do not necessarily need to use `resolve` the paths manually here. `pathlib` will be able to resolve the paths without the call to `resolve`. I will remove it in `check_if_all_paths_exist`.